app/vmui/packages/vmui/src/pages/CustomPanel/InstantQueryTip/InstantQueryTip.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "preact/compat";
+import Hyperlink from "../../../components/Main/Hyperlink/Hyperlink";
+
+const last_over_time = <Hyperlink
+  text="last_over_time"
+  href="https://docs.victoriametrics.com/MetricsQL.html#last_over_time"
+  underlined
+/>;
+
+const instant_query = <Hyperlink
+  text="instant query"
+  href="https://docs.victoriametrics.com/MetricsQL.html#last_over_time"
+  underlined
+/>;
+
+const InstantQueryTip: FC = () => (
+  <div>
+    <p>
+      This tab use {instant_query} that tries to locate a data sample is equal to <b>5m</b>.
```

Comment:
This message is very unclear. It brings more questions than answers :) How above the following message?

```
This tab shows {instant_query} results for the last 5 minutes ending at the selected time range. Please wrap the query into {last_over_time} if you need results over arbitrary lookbehind interval.
```

BTW, is it correct that the tab sets `time` query arg for the [instant_query]() to the `end` time of the selected time range? If not, then please update the tip accordingly.

Reply:
I think this is the correct behavior because if the time query argument is set to the start time of the selected time range, then the displayed data may extend beyond the selected time range.